src/generate/rust.rs

```diff
@@ -16,14 +16,14 @@ pub trait RustInputPat {
     fn rust_matcher(&self) -> Src;
 }
 
-impl<S: AsRef<str>> RustInputPat for ::scannerless::Pat<S> {
```

Comment:
Can you import `crate::scannerless`? I'd prefer it if there were no `crate` keywords except in imports at the top of each file.